tools/darwin/Support/Codesign-topshelf.command

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -x
+
+LDID="$NATIVEPREFIX/bin/ldid"
+
+# Delete existing codesign and provisioning file
+rm -f "${CODESIGNING_FOLDER_PATH}/embedded.mobileprovision"
+rm -rf "${CODESIGNING_FOLDER_PATH}/_CodeSignature"
```

Comment:
Im not sure this should be done in a regular signing environment. Is there a need for this outside of jailbreak usage? I havent come across the same signing issues that forces the need for this in the main app bundle, as xcode manages it all top to bottom for the topshelf appex.

Essentially, should this be inside the fake sign if check below?

Reply:
Also, does CODESIGNING_FOLDER_PATH actually give you the path to the topshelf appex in this context? Or is it only giving you the path to the main app bundle?

Reply:
If I understand correctly your question, this conversation from @kambala-decapitator should answer your question? https://github.com/xbmc/xbmc/pull/18335#discussion_r482011235

Reply:
Concerning `CODESIGNING_FOLDER_PATH`, this is what Jenkins gives:

```
+ rm -f /Users/Shared/jenkins/workspace/TVOS/build/build/Debug-appletvos/kodi-topshelf.appex/embedded.mobileprovision
+ rm -rf /Users/Shared/jenkins/workspace/TVOS/build/build/Debug-appletvos/kodi-topshelf.appex/_CodeSignature
```

So in TopShelf context `CODESIGNING_FOLDER_PATH  == /Users/Shared/jenkins/workspace/TVOS/build/build/Debug-appletvos/kodi-topshelf.appex/`

Reply:
All fine then, if it works